charts/lab-partner/templates/deployment.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: lab-partner
+  namespace: {{ .Values.namespace }}
```

Comment:
Helm will install namespace scoped resources in the namespace the chart is being installed to `helm install --namespace ...` so you shouldn't need add namespace to the metadata. If you do need the namespace value in the chart you can use .Release.namespace https://helm.sh/docs/chart_template_guide/builtin_objects/